types/nightwatch/test/testClientCommands.ts

```diff
@@ -1,7 +1,8 @@
-import { NightwatchAPI } from "nightwatch";
+import { NightwatchAPI, Cookie, NightwatchLogEntry, NightwatchCallbackResult } from "nightwatch";
 
-function isNull(v: null) {}
 function isNightwatchAPI(v: NightwatchAPI) {}
+function isNightwatchCallbackResult<T>(v1: NightwatchCallbackResult<T>, v2: NightwatchCallbackResult<T>) {}
+function isResultType<T>(v1: T, v2: T) {}
```

Comment:
Is there any reason why you need 2 arguments? Only 1 should be enough to check the type.

Reply:
Actually on keeping just 1 argument Typescript was giving an error of unused generic, so we decided sending the type as first argument and the result to be checked as second, but that didn't work out. A better solution for this would be to do something like this:
```suggestion
function isNightwatchCallbackResult<T>(result: NightwatchCallbackResult<T>): T | void {
    if (result.status === 0) {
        return result.value;
    }
}
function isType<T>(v: T): T { return v; }
```

Reply:
interesting, never had that error before. 
Anyway, the solution above looks good 👍